volatility3/framework/plugins/linux/pagecache.py

```diff
@@ -420,8 +420,54 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
         ]
 
     @staticmethod
+    def read_inode_content(
+        inode: inodeClass,
+        buffer: IO,
+        page_size: int,
```

Comment:
The page_size should not be a configurable argument. Using a value different to the layer page_size will lead to incorrect results.